CHANGELOG.md

```diff
@@ -0,0 +1,200 @@
+## Changelog
+
+### 2.21 (Mar 7, 2019)
+
+-   [
+    JENKINS-43194](https://issues.jenkins-ci.org/browse/JENKINS-43194) -
+    Getting issue details... STATUS  Make SCMBinder try to abort builds
```

Comment:
Need to clean up this junk. The exporter tool does not handle certain Confluence macros well.